src/CodeReview.Orchestrator/Utils/CommandLineJsonConvert.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using CommandLine;
+using Newtonsoft.Json;
+
+namespace GodelTech.CodeReview.Orchestrator.Utils
+{
+    public class CommandLineJsonConvert : ICommandLineJsonConvert
+    {
+        public string Serialize(params Type[] type)
+        {
+            if (type is null) throw new ArgumentNullException(nameof(type));
+
+            return JsonConvert.SerializeObject(
```

Comment:
I suggest to use indented output in order to provide nice output. It's a matter of configuring one property.